docs/src/manual/grid.md

```diff
@@ -43,7 +43,13 @@ FerriteMeshParser.get_ferrite_grid
 ```
 
 If you are missing the translation of an Abaqus element that is equivalent to a `Ferrite.Cell`,
-consider to open an [issue](https://github.com/Ferrite-FEM/FerriteMeshParser.jl/issues/new) or a pull request. 
+consider to open an [issue](https://github.com/Ferrite-FEM/FerriteMeshParser.jl/issues/new) or a pull request.
+
```

Comment:
```suggestion
### Reading custom meshes
```
When I checked the docs, it looks like this only applies to FerriteMeshParser otherwise... 
Perhaps you have a better idea for the header naming though...

Reply:
Either yours or double the `pitfalls` , so

```suggestion
### Pitfalls when reading custom meshes
```

Reply:
Yours is more concrete and better!